steps/custom_metrics.py

```diff
@@ -1,9 +1,45 @@
+"""
+This module defines custom metric functions that are invoked during the 'train' and 'evaluate'
+steps to provide model performance insights. Custom metric functions defined in this module are
+referenced by the ``function`` attributes of entries in the ``custom`` subsection of the ``metrics``
+section in ``pipeline.yaml``, for example:
```

Comment:
This is a bit hard to follow. Is this technically correct?

```suggestion
referenced by the ``function`` attributes of entries in the ``metrics.custom``
section in ``pipeline.yaml``, for example:
```

Reply:
I think the `.` notation might be confusing, so I just changed this to

```
referenced in the ``metrics`` section of ``pipeline.yaml``, for example:
```

The example should make pretty clear what we're talking about it

Reply:
Yep I thought that as well. Thanks!